package.json

```diff
@@ -68,6 +81,7 @@
     "vue": "^2.7.14"
   },
   "scripts": {
+    "test": "jest --coverage",
```

Comment:
We shouldn't run coverage by default as that's an expensive operation as the amount of tests grows.